parser/parser.go

```diff
@@ -555,7 +555,7 @@ func (p *parser) validateApp() {
 			astutil.Apply(f.AST, func(c *astutil.Cursor) bool {
 				node := c.Node()
 				if ref, ok := f.References[node]; ok && ref.Type == est.RPCRefNode && !p.validRPCReferences[node] {
-					if _, isCall := c.Parent().(*ast.CallExpr); !isCall {
+					if call, isCall := c.Parent().(*ast.CallExpr); !isCall || call.Fun != node {
```

Comment:
This is a bug fix, which prevents API's being passed as function arguments from being detected.

i.e. we want to prevent `myfunc(mysvc.myAPI)` from being a valid use of `mysvc.myAPI`